env.sh

```diff
@@ -2,15 +2,15 @@
 
 export MACOSX_DEPLOYMENT_TARGET=10.10
 
-export BRAVE_TOR_VERSION="0"
+export BRAVE_TOR_VERSION="1"
```

Comment:
what is this version used for?

Reply:
@diracdeltas - we've already released a version of tor binary with 0.4.5.7, this tracks updates to tor client when the tor version remains the same